xyz_yut_vision_node:
  ros__parameters:
    camera_topic: "/camera/my_d435/color/image_raw"
    camera_info_topic: "/camera/my_d435/aligned_depth_to_color/camera_info"
    depth_camera_topic: "/camera/my_d435/aligned_depth_to_color/image_raw"
    roi_value:
      x: 530
      y: 120
      width: 530
      height: 530
      rotationAngle: 1.3

    Blue_value: [130, 255, 255, 90, 50, 50]
    Pink_value: [170, 255, 255, 150, 50, 50]
    Red_value: [10, 255, 255, 0, 50, 50]
    Black_value: [180, 40, 200, 0, 0, 0]
    Yut_value: [55, 100, 255, 0, 0, 150]
    Orange_value: [25, 255, 255, 10, 100, 150]

    markerCenterOffset:
      x: 0.0
      y: -600.0
      z: 0.0

